src/common/components/Field/Field.tsx

```diff
@@ -0,0 +1,34 @@
+import { type ReactNode } from "react";
+import { WarningCircleIcon } from "@/common/components/CustomIcon";
+import { Label } from "@/common/components/Label";
+import { fieldTheme } from "./Field.theme";
+
+export type FieldProps = {
```

Comment:
You might need to extend ComponentPropsWithoutRef<"div"> here so we can add more stuff like data attributes or className

Reply:
Could be under a separate type so that other components extending it won't have clashes. I usually call them something like FieldFullProps

Reply:
makes sense! updated at [`96f7043`](https://github.com/AfterClass-io/afterclass.io-v2/pull/27/commits/96f70435615135fa77fe2152278bd09a7bd39112)